README.rst

```diff
@@ -15,6 +15,14 @@ command and for specific subcommands through :code:`dcos <subcommand> --help`.
 Additional documentation for the CLI and for the DCOS in general is available
 in the `Mesosphere docs`_.
 
+To run the DCOS CLI as a container, you can pull it from the docker hub with::
+
+    docker run -i -t mesosphere/dcos-cli <url of your cluster>
```

Comment:
cc @pyronicide 

We're trying to decide between two uses of the dcos-cli docker:

1) `docker run -it -v ~/.dcos:/root/.dcos mesosphere/dcos-cli dcos package list`
2)

```
docker run -it mesosphere/dcos-cli
# dcos package list
```

1)
+ you can script and pipe commands from your host
- stupidly long commands.  we'd have to also ship some wrapper (e.g. `dcos-docker`)

2)
+  commands are still the same.  self-contained environment
- config and subcommand changes don't persist
-